shell/src/universal/bin/jupyterlab.sh

```diff
@@ -53,7 +53,7 @@ install_almond() {
       chmod +x coursier
   fi
 
-  SCALA_VERSION=2.13.5 ALMOND_VERSION=0.11.2
```

Comment:
No need to roll back. When we will make a new release, it should be fine. Just keep change in your own fork.

Reply:
Makes sense! Wii change.